src/pages/Home/Home.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import logo from './assets/logo.svg';
+import classes from './styles/Home.module.scss';
```

Comment:
didn't find the setting yet to make this work with .local, the default for react app is .module